assignments/prototype-refactor.js

```diff
@@ -1,9 +1,173 @@
-/* 
+/*
+  === GameObject ===
+  * thiscreatedAt
+  * name
+  * dimensions (These represent the character's size in the video game)
+  * destroy() // prototype method that returns: `${this.name} was removed from the game.`
+*/
+class GameObject {
+  constructor(player) {
```

Comment:
Well done at refactoring your code with classes. Your code is really well written.
I like how meaningful was your parameter name here, a small thing like this make your code even more readable.